src/gcp_scanner/crawl.py

```diff
@@ -728,8 +724,8 @@ def get_filestore_instances(project_id: str,
         parent=f"projects/{project_id}/locations/-")
     while request is not None:
       response = request.execute()
-      for instance in response.get("instances", []):
-        filestore_instances_list.append(instance)
+      filestore_instances_list = [instance 
+                                  for instance in response.get("instances", [])]
```

Comment:
Redundant.

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.